spacy/displacy/render.py

```diff
@@ -247,6 +247,15 @@ def __init__(self, options={}):
         self.direction = DEFAULT_DIR
         self.lang = DEFAULT_LANG
 
+        template = options.get("template", "")
```

Comment:
Not a big deal, but we might as well just make it default to `None` here:

```suggestion
        template = options.get("template")
```